src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/SetAttributeCommand.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.References
+{
+    internal class SetAttributeCommand : IProjectSystemUpdateReferenceOperation
+    {
+        private readonly ConfiguredProject _selectedConfiguredProject;
+        private readonly string _itemSpecification;
+        private readonly AbstractReferenceHandler _referenceHandler;
+
+        public SetAttributeCommand(AbstractReferenceHandler abstractReferenceHandler, ConfiguredProject selectedConfiguredProject, string itemSpecification)
+        {
+            _referenceHandler = abstractReferenceHandler;
+            _selectedConfiguredProject = selectedConfiguredProject;
+            _itemSpecification = itemSpecification;
+        }
+
+        public async Task<bool> ApplyAsync(CancellationToken cancellationToken)
+        {
+            IProjectItem item = await GetProjectItem();
+
+            if (item == null)
+            {
+                return false;
+            }
+
+            await item.Metadata.SetPropertyValueAsync(ProjectReference.TreatAsUsedProperty, PropertySerializer.SimpleTypes.ToString(true), null);
```

Comment:
>PropertySerializer.SimpleTypes.ToString(true) [](http://example.com/codeflow?start=92&length=45)

Would it make sense to precompute these in the constructor to avoid repetitive work later?

Reply:
done